### DISCUSSION TITLE
Images not loading after process restart

### DISCUSSION THREAD

**Discussion by ghost** [requarks/wiki#3336] [Error / Bug Report]
**Describe the bug**
When hosting the application with PM2, the images fail to load once the process has been restarted. It just shows the name of the image on the page. 

**To Reproduce**
1. install pm2 
2. Change your directory to the application
3. use pm2 start ./server/index.js --name Wiki.js
4. upload an image 
5. insert it into the page and save it 
6. restart the process with pm2 (pm2 restart Wiki.js) 
7. View the page - the image is now only the name 

**Expected behavior**
The image should still render after restart and re-uploads should not be necessary, 

**Screenshots**

![image](https://user-images.githubusercontent.com/44342932/109470810-93d7f500-7a78-11eb-84db-69af312a2e65.png)

**Host Info (please complete the following information):**
 - OS: Ubuntu Linux 18.04.2 
 - Wiki.js version: 2.5.170
 - Database engine: MySQL 8.0.23

**Comment by ghost** [comment]
Hi there. Is this issue going to be resolved?

**Comment by NGPixel** [comment]
I can't replicate your issue.

**Comment by phamkhanh07** [comment]
im facesing same error, when view image in new tab "Internal Server Error"

**Comment by xBlizZer** [comment]
I'm facing the same or nearly the same error. In my case, the images also don't show up. 
![image](https://user-images.githubusercontent.com/33377032/116402318-8f3b7d00-a82c-11eb-99d5-c213ecf927f1.png)

My console gave me this error:
![image](https://user-images.githubusercontent.com/33377032/116402391-ad08e200-a82c-11eb-8604-753900b08492.png)

When I try to view the image in a new tab I get an "Internal Server Error"
If I reupload the image, the uploaded image shows perfectly fine.

Information: 
DB: MSSQL
Wiki Version: 2.5.201
NodeJS Version: 14.16.1

**Reply by ghost** [reply]
I actually managed to solve this issue. First, make sure that the images you saved are in the right directory and that you are referencing them properly in the editor. 

If they still don't work, then you need to reinstall the wiki. I did this and it worked fine for me.

**Reply by icf-hamburg-admin** [reply]
Do you mean by reinstalling, removing all data and starting from scratch? Or is it possible to reinstall without losing the data?

**Comment by icf-hamburg-admin** [comment]
I have the same issue. For me, images work for a while and then after some time they can not be loaded anymore: Internal Server Error. Probably after the instance was restarted.
